src/coresyf_image_crop/coresyf_image_crop.py

```diff
@@ -69,6 +68,47 @@ def get_shapefile_crs(data_source):
     return epsg_code
 
 
+def get_raster_crs(data_source):
+    '''
+    Retrieves the ESPG Code of the raster CRS (Coordinate Reference System).
+
+    data_source: must be the result of GDAL Open applied to the respective GDAL
+    driver.
+    '''
+    try:
+        projection = data_source.GetProjectionRef()
+        ident = Sridentify(prj=projection)
+        epsg_code = ident.get_epsg()
+    except Exception as epsg_exception:
+        print(epsg_exception)
```

Comment:
An exception message isn't, in general, something we expect a _stdout_ of a robust program. What would you think of a program were something like` 'None' object has no attribute '.GetProjectionRef()'` appears in the _stdout_. :stuck_out_tongue_winking_eye: